CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ## [Unreleased]
```

Comment:
The URL backing this header needs to be updated.

Reply:
Fixed in 110908e